requirements-dev.txt

```diff
@@ -52,3 +52,6 @@ fastparquet<2023.1.0
 # for release script
 PyGithub
 pygit2
+moto
+flask-cors
```

Comment:
What is this package for? Should we also add it into other env files?

Reply:
Looks like this is an inconsistency issue of `moto` packages in PyPI and conda-forge. Would you mind opening an issue in https://github.com/getmoto/moto, for instance?